homeassistant/components/enphase_envoy/strings.json

```diff
@@ -41,7 +41,8 @@
       "init": {
         "title": "Envoy {serial} {host} options",
         "data": {
-          "diagnostics_include_fixtures": "Include test fixture data in diagnostic report. Use when requested to provide test data for troubleshooting or development activies. With this option enabled the diagnostic report may take more time to download. When report is created best disable this option again."
+          "diagnostics_include_fixtures": "Include test fixture data in diagnostic report. Use when requested to provide test data for troubleshooting or development activies. With this option enabled the diagnostic report may take more time to download. When report is created best disable this option again.",
+          "disable_keep_alive": "Always use a new connection when requesting data from the Envoy. May resolve communication issues with some Envoy firmwares. After changing this option, reload the integration."
```

Comment:
Instead of telling a user to reload, we can also just reload directly :)

Add this to your `__init__.py`:

```py
async def async_reload_entry(hass: HomeAssistant, entry: ConfigEntry) -> None:
    """Reload the config entry when it changed."""
    await hass.config_entries.async_reload(entry.entry_id)
```

And call this on the `async_setup_entry` to listen for changes and kick off the reload:

```py
    # Reload entry when it is updated.
    entry.async_on_unload(entry.add_update_listener(async_reload_entry))
```

Voila!

Reply:
Thanks for the tip @frenck, I'll add it.